### PR TITLE
More detailed language pack extension description

### PR DESCRIPTION
**Pull request by nickzhums**
By looking at the description of other language extensions, our java extension pack description is quite simple, so adding a few more details to provide more clarity for developers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "onWebviewPanel:java.welcome",
     "onCommand:java.classpathConfiguration",
     "onWebviewPanel:java.classpathConfiguration",
+    "onWebviewPanel:java.markdownPreview",
```
Comment (Eskibear) [inline package.json:58]: what's this line for? @jdneo please take a look

Comment (nickzhums) [reply package.json:58]: i don't think i edited this line but somehow it's showing up here, i can remove it if helps

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "vscode-java-pack",
   "displayName": "Java Extension Pack",
-  "description": "Popular extensions for Java development and more.",
+  "description": "Popular extensions for Java development that provides Java intellisense, debugging, testing, Maven support, project management and more",
```
Comment (testforstephen) [inline package.json:4]: intellisense -> IntelliSense. It's a general term. See https://code.visualstudio.com/docs/editor/intellisense

Maven support -> Maven/Gradle support

**Review by jdneo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jdneo) [reply package.json]: This line has been removed in a recent commit. Feel safe to remove it

Comment (nickzhums) [reply package.json]: just removed

**Comment by nickzhums** [comment]
anyone can help to merge this PR? I don't have write access : ( thanks~

**Comment by jdneo** [comment]
Thank you @nickzhums

**Pull request merged**